antareslauncher/use_cases/retrieve/download_final_zip.py

```diff
@@ -28,7 +28,7 @@ def download(self, study: StudyDTO):
             The updated data transfer object, with its `local_final_zipfile_path`
             attribute set if the download was successful.
         """
-        if study.finished and not study.with_error and not study.local_final_zipfile_path:
```

Comment:
Here too we should probably keep the condition on the zip path to not download it twice (the next time the "launcher" is executed)